docs/src/Button.doc.js

```diff
@@ -102,12 +102,12 @@ card(
         href: 'iconEnd',
       },
       {
-        name: 'inline',
+        name: 'fullWidth',
         type: 'boolean',
         required: false,
-        defaultValue: false,
+        defaultValue: true,
```

Comment:
In the long-term, I don't think we want this to default to "true". Is the plan to invert this default value and codemod existing usages at some point?

Reply:
'm changing from false to true because `inline` is the opposite of `fullWidth`. 

If Button had inline='false' <Button/> means than now it must be fullWidth='true' <Button />
inline='false' becomes fullWidth='false'

I'm keeping the same logic to prevent major changes and points of failure.

Reply:
Right, but I believe the whole point of this change was to update the default behavior. Changing the prop name is more of a side effect to keep the API consistent.

In general, we try to avoid having default boolean props of "true"; it's a cleaner API to have the default behavior be "false" and for the user to pass a boolean prop to indicate the non-default behavior (e.g. how `inline` currently works).

The motivation behind this change was that the vast majority of the time we don't want full-width buttons, so having that as the default state is annoying. The default state should represent the most common usage.

This change should be safe with the proper codemod. In the end, most buttons should be something like `<Button {...otherProps} />` and only full-width buttons should be `<Button fullWidth {...otherProps} />`.

Reply:
Got it... didn;t read the ticket content. You mentioned that in there. Cool. I'll change it, it does makes sense.

Reply:
@dangerismycat Fixed. Thanks for the feedback.